addons/point_of_sale/static/src/app/screens/payment_screen/payment_screen.js

```diff
@@ -82,11 +82,15 @@ export class PaymentScreen extends Component {
     get selectedPaymentLine() {
         return this.currentOrder.get_selected_paymentline();
     }
-    async addNewPaymentLine(paymentMethod) {
+    async addNewPaymentLine(paymentMethod, automatic_payment) {
         // original function: click_paymentmethods
-        const result = this.currentOrder.add_paymentline(paymentMethod);
+        const result = this.currentOrder.add_paymentline(paymentMethod, automatic_payment);
         if (result) {
             this.numberBuffer.reset();
+            if (automatic_payment) {
```

Comment:
```suggestion
            if (paymentMethod.use_payment_terminal) {
```

remove the other diffs